Content.Tests/DMProject/Tests/Regex/regex_L_brackets.dm

```diff
@@ -0,0 +1,5 @@
+
+/proc/RunTest()
+	var/static/regex/repeated_consonant_regex = regex(@"\b([^aeiou\L])\1", "gi")
+	var/words = "hey ggggurl"
+	ASSERT(repeated_consonant_regex.Find(words) == 5)
```

Comment:
This would match the same regardless of if `\L` wasn't in the regex. Add something that changes the outcome depending on if `\L` is there.
```suggestion
	var/words = "hey... ggggurl"
	ASSERT(repeated_consonant_regex.Find(words) == 8)
```